### ISSUE TITLE
All ops don't work

### ISSUE THREAD

**Issue by dzfranklin** [peterldowns/pgmigrate#10]
All ops fail with the same error

```
> pgmigrate -d postgres://localhost:5432/notes ops recalculate-checksum --all


error: failed to set checksum: ERROR: relation "public.pgmigrate_migrations" does not exist (SQLSTATE 42P01)
```

But other commands work, for example

```
>  pgmigrate -d postgres://localhost:5432/notes verify
```

v0.1.0+commit.7f13363 installed via homebrew

**Comment by peterldowns** [comment]
@dzfranklin working on it, will have a PR soon. It's because of the changes I made to fully qualify the migrations table name in #7.

**Comment by peterldowns** [comment]
@dzfranklin done, I've tagged and released v0.2.0 and [updated the homebrew tap](https://github.com/peterldowns/homebrew-tap/commit/87e2d2dc0f2016f54536fe188b7d2f1ef7c5babc). You should be able to upgrade and see the commands work again. Thanks for reporting this issue, and sorry for the hassle --- bit embarassing that I didn't have tests on the `ops` commands until now.
